src/test/groovy/org/gradle/android/ConfigurationCachingTest.groovy

```diff
@@ -0,0 +1,21 @@
+package org.gradle.android
+
+class ConfigurationCachingTest extends AbstractTest {
+    def "Can run with configuration cache"() {
+        given:
+        SimpleAndroidApp.builder(temporaryFolder.root, cacheDir)
+                .withAndroidVersion(Versions.latestAndroidVersion())
+                .withKotlinDisabled()
+                .build()
+                .writeProject()
+
+        when:
+        def result = withGradleVersion(Versions.latestGradleVersion().version)
+                .withProjectDir(temporaryFolder.root)
+                .withArguments('--configuration-cache', 'assembleDebug')
+                .build()
+
+        then:
+        !result.output.contains("problems were found storing the configuration cache")
```

Comment:
Should we also run an additional execution and check that the configuration cache is reused on a follow up execution?  See https://github.com/gradle/test-retry-gradle-plugin/blob/master/plugin/src/test/groovy/org/gradle/testretry/ConfigCachingPluginFuncTest.groovy for an example.